### PR TITLE
fix: ExpressionChangedAfterItHasBeenCheckedError

### LINKED ISSUE #93: Fixer l'erreur ExpressionChangedAfterChecked de la composante Bao-Button
bao-button_error

Ce message d'erreur apparaît dans toutes les applications qui utilisent la composante bao-button, en développement local seulement.

### PR DESCRIPTION
**Pull request by mtlealr**
close #93

### CODE REVIEW

File: projects/angular-ui/src/lib/button/button.component.ts [hunk 1]
```diff
@@ -97,13 +97,18 @@ export class BaoButtonComponent implements AfterViewInit {
   public ngAfterViewInit() {
```
Comment (OlivierAlbertini) [inline projects/angular-ui/src/lib/button/button.component.ts:97]: Thanks for submitting this PR. Did you try to replace ngAfterViewInit by ngAfterContentInit

Comment (mtlealr) [reply projects/angular-ui/src/lib/button/button.component.ts]: This is indeed a better solution. I tested it and it works well. I updated the PR

File: RELEASING.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Run:
 npm version --no-git-tag-version x.y.z
 cd projects/angular-ui
 npm version x.y.z
-git tag vx.y.z
+git commit -s -am "ci(release): vx.y.z"
```
Comment (OlivierAlbertini) [inline RELEASING.md:17]: I think that you take another commit by mistake ?

File: projects/angular-ui/src/lib/button/button.component.ts [hunk 3]
```diff
@@ -94,7 +94,7 @@ export class BaoButtonComponent implements AfterViewInit {
     return this.elementRef.nativeElement;
   }
 
-  public ngAfterViewInit() {
+  public ngAfterContentInit() {
```
Comment (OlivierAlbertini) [inline projects/angular-ui/src/lib/button/button.component.ts:97]: I'm glad you make it works !

**Review by OlivierAlbertini — COMMENTED** [review]
Please fix the releasing.md extra file and we are good to go, if you can squash in order to make 1 commit instead of 3, it would be perfect. 

Thanks

**Comment by mtlealr** [comment]
rebase and squash done

**Pull request merged**